.github/workflows/performance.yml

```diff
@@ -1,8 +1,9 @@
 name: Performances Tests
 
-on: [pull_request]
-  paths-ignore:
-  - '**.md'
+on:
+  pull_request:
+    paths-ignore:
+    - '**.md'
```

Comment:
I think all of these list items need to be indented one step further. That's how it looks in the [GitHub docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration), anyway. Not sure if it makes a syntactical difference.

Reply:
I thought so too, but we don't do that with any of the `steps` on any of the workflows, so I don't think it matters. Also the YAML linter built into VSCode didn't care.